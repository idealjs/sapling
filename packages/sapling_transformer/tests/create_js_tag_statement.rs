use sapling_transformer::compatible::create_js_tag_statement::generate_create_js_tag_statement;

#[test]
fn test_generate_create_js_tag_statement() {
   let stmt1 = generate_create_js_tag_statement("_el1$", "div");
   let stmt2 = generate_create_js_tag_statement("_el2$", "div");
   insta::assert_snapshot!(format!("{}\n{}", stmt1.to_string(), stmt2.to_string()));
}
