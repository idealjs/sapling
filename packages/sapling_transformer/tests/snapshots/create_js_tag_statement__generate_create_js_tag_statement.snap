---
source: packages/sapling_transformer/tests/create_js_tag_statement.rs
expression: "format!(\"{}\\n{}\", stmt1.to_string(), stmt2.to_string())"
---
let _el1$ = _$createElement("div");
let _el2$ = _$createElement("div");
